tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -297,9 +423,18 @@ declare var cfg:ISnippetConfig;
                     // IE10, Opera calls loaded before the script is processed.
                     // so delaying to give the script a chance to be processed
                     setTimeout(function() {
-                        if (isAbort || !appInsights.core) {
-                            _handleError();
+                        if (isAbort){
+                            if (isOneDS){
```

Comment:
extension of above comment
- This stuff probably needs to remain here though as its only a few lines and uses the current functions
- So we only need to "move" the large (mostly) stand alone support functions (if we can)